Mono.Addins/Mono.Addins.Database/AddinDatabase.cs

```diff
@@ -1826,9 +1826,9 @@ ISetupHandler GetSetupHandler ()
 //			if (Util.IsMono)
 //				return new SetupProcess ();
 //			else
-			if (fs.RequiresIsolation)
-				return new SetupDomain ();
-			else
+			//if (fs.RequiresIsolation)
+			//	return new SetupDomain ();
```

Comment:
This creates an isolated domain to load the files containing mono.addins extensions and generates the database.

In example, in SetupLocal, it could technically fail due to Mono.Cecil being loaded by mono-addins and by an extension and having different versions.

Reply:
It should now use SetupProcess if isolation is required.